web/controllers/api/owner_controller.ex

```diff
@@ -1,19 +1,21 @@
 defmodule HexWeb.API.OwnerController do
   use HexWeb.Web, :controller
+  alias HexWeb.Owner
```

Comment:
alternatively this could go to `HexWeb.Web.controller` although it's unlikely one controller would use more than 1 of these modules.